assets/js/cell/live_editor.js

```diff
@@ -53,6 +53,23 @@ class LiveEditor {
       this._onCursorSelectionChange &&
         this._onCursorSelectionChange(event.selection);
     });
+
+    this.editor.addAction({
```

Comment:
Could you move this setup to `__mountEditor`? That should be the best place :)

Reply:
Done.